rustls/src/webpki/client_verifier.rs

```diff
@@ -135,7 +139,7 @@ impl ClientCertVerifierBuilder {
         mut self,
         supported_algs: WebPkiSupportedAlgorithms,
     ) -> Self {
-        self.supported_algs = Some(supported_algs);
+        self.supported_algs = supported_algs;
```

Comment:
Should we just ditch this method and require the caller to build a custom `CryptoProvider`? Seems pretty niche.

Reply:
Have done that.